modules/travel_pay/app/services/travel_pay/claims_service.rb

```diff
@@ -11,27 +11,21 @@ def get_claims(veis_token, btsss_token, params = {})
       {
         data: claims.map do |sc|
           sc['claimStatus'] = sc['claimStatus'].underscore.titleize
+          sc['id'] = Digest::SHA256.hexdigest(sc['id'])
           sc
         end
       }
     end
 
     def get_claim_by_id(veis_token, btsss_token, claim_id)
-      # ensure claim ID is the right format, allowing any version
-      uuid_all_version_format = /^[0-9A-F]{8}-[0-9A-F]{4}-[0-9A-F]{4}-[89ABCD][0-9A-F]{3}-[0-9A-F]{12}$/i
-
-      unless uuid_all_version_format.match?(claim_id)
-        raise ArgumentError, message: "Expected claim id to be a valid UUID, got #{claim_id}."
-      end
-
```

Comment:
assuming this approach is appropriate for anonymizing ids, do we want to keep this logic but update it for a 64-character hexdigest?